sale_stock_line_customer_ref/models/sale_order_line.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    customer_ref = fields.Char(string="Customer Ref.")
```

Comment:
For [consistency with the core field in `sale.order`](https://github.com/odoo/odoo/blob/2cd813c5911c4eb55473e39fa071370c2148143c/addons/sale/models/sale.py#L161), maybe consider naming it `client_order_ref`

Reply:
I prefer to keep `customer_ref` :thinking: and not overlapping the `client_order_ref` std field.

Reply:
Well it wouldn't overlap as one is for sale.order and the new one is for the lines. As the purpose seems to be the same, maybe the name could be the same.
But anyway, not blocking 😉